packages/map-popup/i18n/en-US.yml

```diff
@@ -3,7 +3,8 @@ otpUi:
     availableBikes: "Available bikes: {value}"
     availableDocks: "Available docks: {value}"
     floatingBike: "Free-floating bike: {name}"
-    floatingCar: "{company} {name}"
+    floatingCar: "Car: {name}"
```

Comment:
Is the colon needed?

Reply:
Without the colon things can get messy especially if the company name is silly